includes/polling/functions.inc.php

```diff
@@ -631,7 +631,7 @@ function ($carry, $metric_group) use ($metrics) {
                     'application_metrics'
                 );
                 echo '+';
-            } elseif ($value != $db_metrics[$metric_name]['value']) {
+            } elseif ($value != $db_metrics[$metric_name]['value'] || Str::is(Config::get('applications.metric.always_update'), $metric_name)) {
```

Comment:
Shouldn't it be in_array?

Reply:
It works with `Str::is` (I've tested with 1 and 2 elements in the array)